watchtower/src/notifier.rs

```diff
@@ -8,11 +8,19 @@ struct TelegramWebHook {
     chat_id: String,
 }
 
+struct TwilioWebHook {
+    twilio_account: String,
```

Comment:
nit: within the `TwilioWebHook` struct, it seems unnecessary to prefix each member with `twilio_`

Reply:
i can change